Sources/KNContactsPicker/KNContactsPickerController.swift

```diff
@@ -80,6 +83,17 @@ class KNContactsPickerController: UITableViewController {
         return Array(selectedContacts)
     }
     
+    @objc func selectAllSelected() {
+        let sections = tableView.numberOfSections
+        for section in 0..<sections {
+            let rows = tableView.numberOfRows(inSection: section)
+            for row in 0..<rows {
+                let contact = self.getContact(at: IndexPath(row: row, section: section))
+                self.toggleSelected(contact)
+            }
+        }
```

Comment:
I think instead of iterating over all of these sections and rows, I would go with setting the `selectedContacts` directly. 

Something along the lines of: 
```swift
let contactsToAdd = isFiltering ? filteredContacts : contacts
selectedContacts = Set(contactsToAdd)
```

Reply:
Fixed it